packages/cli/src/util/dev/server.ts

```diff
@@ -931,7 +931,7 @@ export default class DevServer {
     }
 
     this._address = new URL(
-      address.replace('[::]', 'localhost').replace('127.0.0.1', 'localhost')
+      address.replace('[::]', 'localhost').replace('0.0.0.0', 'localhost')
```

Comment:
Should we replace both 127.0.0.1 and 0.0.0.0?

Reply:
I don't think so, because the only case where it will be 127.0.0.1 is if the user explicitly specified that, in which case we should show the explicit value.

Reply:
Ok seems good, just need a test